apps/web/src/app/[locale]/page.tsx

```diff
@@ -70,7 +70,7 @@ export default async function Home({
           {announcement ? <AnnouncementCard news={announcement} /> : null}
         </div>
         <section className="space-y-4">
-          <h1 className="font-mono text-4xl font-bold text-gray-900">
+          <h1 className="dark:text-dark-heading font-mono text-4xl font-bold text-gray-900">
```

Comment:
This is a bigger note and imo not in scope of this PR, but we could move toward replacing straight color tokens like `gray-900` with more semantic tokens overall like `primary-text` or similar.

Would also make it easier to assign those tokens in the tailwind theme and avoid having to define both `dark:` and light colors separately, since `primary-text` could already have both variants.

Inspired by [shadcn](https://ui.shadcn.com/docs/installation/manual)

Reply:
This would also make the dark mode (or light mode, whichever is not the default) less error prone